src/getHumanAge.test.js

```diff
@@ -3,8 +3,50 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return zeros if getHumanAge(0, 0)`, () => {
+    expect(getHumanAge(0, 0))
+      .toStrictEqual([0, 0]);
+  });
+
+  test(`should return zeros if getHumanAge(14, 14)`, () => {
```

Comment:
The test description and the expected output do not match. The test description says it should return zeros but the expected output is [0, 0]. This is ambiguous and might confuse other developers.